src/cosmoz-bottom-bar-focused-context.js

```diff
@@ -0,0 +1,26 @@
+import { createContext, useContext, useState, useMemo } from '@pionjs/pion';
+import { noop } from '@neovici/cosmoz-utils/function';
+
+export const bottomBarFocused = createContext({
+	bars: [],
+	add: noop,
+	remove: noop,
+});
+customElements.define('bottom-bar-focused-provider', bottomBarFocused.Provider);
+
+export const useBottomBarFocusedCtx = () => useContext(bottomBarFocused) ?? {};
+
+export const useBottomBarFocusedCtxHandler = () => {
```

Comment:
```suggestion
export const useFocusHandler = () => {
```

Reply:
Remove bottomBar prefix here even if I'll import it in cosmoz-frontend?